### DISCUSSION TITLE
WARNING: could not locate iTunesMetadata.plist in archive!

### DISCUSSION THREAD

**Discussion by markkelynack** [airsdk/Adobe-Runtime-Support#723] [General]
Hi All, Just back to working on iOS apps, and getting this error when installing the Dev version on phone.

(WARNING: could not locate iTunesMetadata.plist in archive!)

iOS 13.4 and iOS 14.4, both fail to install - SDK AIR_33.1.1.406

Did this ever get solved, would love some help on what to look for, google has come up short.

Cheers

MK

**Comment by marchbold** [comment]
Have you updated the adt.jar from here: 

https://transfer.harman.com/message/qhYaztL37nVk3buy1EHv61

There's a few signing issues and things under development with 406 around the change of certs from Apple.

**Reply by markkelynack** [reply]
Brilliant, worked a treat, thank you

**Reply by tommy-banana** [reply]
Any chance of getting an up to date link to that? Having same issue...

**Reply by ajwfrost** [reply]
@tommy-banana the new ADT.jar is now part of our 33.1.1.444 SDK so you can download it from the website and will get the fixes for code-signing.
thanks

**Comment by thatkidrich** [comment]
I am getting the same error trying to load an app onto my iOS device from Animate CC 2021. Is there a fix for this? I have downloaded the latest SDK from HARMAN.

**Reply by ajwfrost** [reply]
@thatkidrich could you please provide the full error details/output, as the "could not locate iTunesMetadata.plist" report is just a warning and does not mean that the operation didn't complete..
thanks

**Comment by xxxVBxxx** [comment]
I'm getting the same error when trying to install directly on a device.

```
WARNING: could not locate iTunesMetadata.plist in archive!
WARNING: could not locate Payload/MyApp.app/SC_Info/MyApp.sinf in archive!
ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008016: Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.YlkHWi/extracted/Payload/MyApp.app : 0xe8008016 (The executable was signed with invalid entitlements.)
```

**Reply by ajwfrost** [reply]
Hi

Which version of AIR SDK are you using for this? The warnings you can ignore but the errors here are showing it's either got an invalid provisioning profile, or it's got the wrong entitlements. So it may be things like a provisioning profile for the app store being used for ad hoc, or vice versa, or developer certificates being used instead of distribution ones..

thanks

**Reply by xxxVBxxx** [reply]
I forgot to switch aps-environment from production to development, it's working now, sorry for bothering you

```
<key>aps-environment</key>
<string>development</string>
```

**Reply by ajwfrost** [reply]
Ah - yes I wish there was a better way of checking a lot of these various different dependencies so that's it's easeir to spot any disparities and conflicts between the certificate/profile/entitlements/etc!

thanks
